internal/cli/universal_login_customize.go

```diff
@@ -20,14 +22,21 @@ import (
 )
 
 const (
-	webAppURL                = "http://localhost:5173"
+	webServerPort            = "52649"
+	webServerURL             = "localhost:" + webServerPort
+	webServerURLWithHTTP     = "http://localhost:" + webServerPort
```

Comment:
Maybe call `webServerURL` -> `webServerHost`, and `webServerURLWithHTTP` -> webServerURL`?